docs/clients/hobby_input.rst

```diff
@@ -109,10 +109,18 @@ Message Table
 
 Type ID 76 | Hobby Input
 
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
-| Sub ID |    Short Name     |     Access     | Data Type | Unit |                              Note                               |
-+========+===================+================+===========+======+=================================================================+
-| 0      | allowed_protocols | get, set, save | uint8     | enum | Standard PWM = 1, OneShot125 = 2, OneShot42 = 3, MultiShot = 4, |
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
-| 1      | protocol          | get, set, save | uint8     | enum |                                                                 |
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| Sub ID | Short Name               | Access         | Data Type | Unit | Note                                                            |
++========+==========================+================+===========+======+=================================================================+
+| 0      | allowed_protocols        | get, set, save | uint8     | enum | Standard PWM = 1, OneShot125 = 2, OneShot42 = 3, MultiShot = 4, |
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| 1      | protocol                 | get, set, save | uint8     | enum |                                                                 |
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| 2      | calibrated protocol      | get            | uint8     | enum |                                                                 |
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| 3      | calibrated_high_ticks_us | get, set, save | uint8     | enum |                                                                 |
```

Comment:
Description: "The number of microseconds of the calibrated protocol considered to be a 100% throttle command"

Units are microseconds